manuscript/chapter1.md

```diff
@@ -747,36 +721,34 @@ class App extends Component {
 }
 ~~~~~~~~
 
-The App class extends from `Component`. Basically, when you declare the App component it extends from another component. What does extend mean? In object-oriented programming, there is the principle of inheritance, which means that functionalities can be passed from one class to another class.
-
-The App class extends functionality from the Component class. To be more specific, it inherits functionalities from the Component class. The Component class is used to extend a basic ES6 class to a ES6 component class. It has all the functionalities that a component in React needs to have. The render method is one of these functionalities that you have already used. You will learn about other component class methods later on.
+When you declare the App component it extends from another component. In object-oriented programming, the term "extends" refers to the principle of inheritance, which means that functionality can be passed from one class to another. The App class extends from the Component class, meaning it inherits functionality from the Component class. The Component class is used to extend a basic ES6 class to a ES6 component class. It has all the functionalities that a component in React needs. The render method is one function you have already used. You will learn about other component class methods as we move along.
 
-The `Component` class encapsulates all the implementation details of a React component. It enables developers to use classes as components in React.
+The `Component` class encapsulates all the implementation details of a React component, which allows developers to use classes as components in React.
 
-The methods exposed by a React `Component` are its public interface. One of these methods must be overridden, while the others don't need to be overridden. You will learn about the latter ones when the book arrives at lifecycle methods in a later chapter. The `render()` method has to be overridden, because it defines the output of a React `Component`. Therefore, it must be defined.
-
-Now you know the basics around JavaScript ES6 classes and how they are used in React to extend them to components. You will learn more about the Component methods when the book describes React lifecycle methods.
+Methods exposed by a React `Component` are its public interface. One of these methods must be overridden, while the others don't need to be overridden. You will learn about these when we discuss lifecycle methods later. The `render()` method has to be overridden, because it defines the output of a React `Component, so it must be defined. These are the basics of JavaScript ES6 classes, and how they are used in React to extend them to components. 
```

Comment:
one backtick after "... because it defines the output of a React `Component" missing